src/foam/nanos/export/GoogleSheetsOutputter.js

```diff
@@ -89,8 +89,14 @@ foam.CLASS({
               propValues.push(obj[columnMethadata[i].columnName].toString().substring(0, 8));
               columnMethadata[i].perValuePatternSpecificValues.push(obj[columnMethadata[i].columnName].toString().substring(8));
             }
-            else
-              propValues.push(obj[columnMethadata[i].columnName].toString());
+            else {
```

Comment:
Why not and 'else if' instead of an 'else' and then an 'if'?

Reply:
Fixed